src/components/Icons/index.tsx

```diff
@@ -50,6 +51,7 @@ export const DownCaret = (
         style={{ width: "16px", transform: "rotate(180deg)" }}
     />
 );
+export const FilledCaret = <CaretDownFilled />;
```

Comment:
1) the caret can be rotated so isn't always pointing down. 2) slightly more readable since all the other icons are declared/renamed in this way. Neither reason is obligatory. We could export and use `{CaretDownFilled}` directly.